build.gradle

```diff
@@ -12,36 +11,26 @@ ext {
     commonsCsvVersion = '1.9.0'
     commonsIoVersion = '2.11.0'
     guavaVersion = '30.1-jre'
+    javaxJsonVersion = '1.1.4'
+    kelpieVersion = '1.2.3'
     picoCliVersion = '4.1.4'
```

Comment:
We not longer need this `picoCliVersion`?

Reply:
Thank you for the good catches. Fixed in 2286bc9.